src/convolution_api.cpp

```diff
@@ -56,16 +56,8 @@ static inline auto MakeFwdCtxAndProblem(miopenHandle_t handle,
     const auto direction =
         (conv.mode != miopenTranspose) ? Direction::Forward : Direction::BackwardData;
 
-    auto problem = (conv.mode != miopenTranspose) ? ProblemDescription{miopen::deref(xDesc),
-                                                                       miopen::deref(wDesc),
-                                                                       miopen::deref(yDesc),
-                                                                       conv,
-                                                                       direction}
-                                                  : ProblemDescription{miopen::deref(yDesc),
-                                                                       miopen::deref(wDesc),
-                                                                       miopen::deref(xDesc),
-                                                                       conv,
-                                                                       direction};
+    auto problem = ProblemDescription{
+        miopen::deref(xDesc), miopen::deref(wDesc), miopen::deref(yDesc), conv, direction};
```

Comment:
See https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2476#issuecomment-1775390668

Reply:
@atamazov Is there any alternative we can compare as a quick patch fix?

Reply:
@junliume Fix is done, in local testing. ETA for PR is tomorrow morning